### ISSUE TITLE
Captchas are annoying. Possible solutions provided.

### ISSUE THREAD

**Issue by lffin1** [aviaryan/Kissanime-Batch-Downloader#30]
So, at the moment Kissanime.ru has a Captcha system that makes it absolute HELL to set up a Bulk Download for an entire series at the moment. This could be bypassed by having the script "solve" every Captcha by skipping it in favor of the RapidVideo host.

There is an alternative website which seems to have the new episodes but an old layout and NO Captchas. This alternative is Kissanime.ac. I suggest setting the Script up for this one because any IDM download from this site is SUPER fast!
<img width="957" alt="kissanimeexample" src="https://user-images.githubusercontent.com/41711979/43236501-c88b681e-90c8-11e8-8416-0dacf4b665b4.png">

**Comment by aviaryan** [comment]
You are right. `Kissanime.ac` looks great. Would you like to send a PR for the same?
I lost hope when they added so many captchas.

**Comment by 9inpachi** [comment]
I took a look at it and it seems Kissanime.ac doesn't have any download links.
Alternatively, I tried to edit the script for www.kiss-anime.ws but later found that the download links there are dead too.

**Comment by Bubbet** [comment]
Kissanime.ac just clones the hydrax server three times and hydrax is a nightmare to find the video link for.
